setup_board

```diff
@@ -265,6 +265,9 @@ PORTAGE_BINHOST="${BOARD_BINHOST}"
 # You can use --select to override this.
 EMERGE_DEFAULT_OPTS="--oneshot"
 
+# SELinux installed policies.
+POLICY_TYPES="mcs"
```

Comment:
Is there a reason to keep `POLICY_TYPES` in the base profile set to `targeted mcs mls`?  Maybe we should make this change there instead.  Or, if we don't want it in the base policy, maybe we can set this value in `profiles/coreos/targets/generic/make.defaults` so that there is only one file with this change when using multiple boards.

Reply:
I missed that entry in the base profile.  I think `profiles/coreos/targets/generic/make.defaults` is the right place to set it since we are only building the mcs policy, and that is the same for all boards.  I'll move this change there.